crates/bevy_render/src/entity.rs

```diff
@@ -106,7 +110,11 @@ impl OrthographicCameraBundle {
                 name: Some(base::camera::CAMERA_3D.to_string()),
                 ..Default::default()
             },
-            orthographic_projection: Default::default(),
+            orthographic_projection: OrthographicProjection {
+                scaling_mode: ScalingMode::FixedVertical,
+                depth_calculation: DepthCalculation::Distance,
```

Comment:
This should already be the default.

Reply:
I know. I added it explicitly, just to ensure that upon any possible future changes, it does not accidentally break. I don't think there is any harm in explicitly overriding it with the correct value. It just seems safer (and more sensible) to me that the camera bundle constructors should explicitly ensure that any relevant parameters are set to the correct values, rather than implicitly relying on the default just happening to be the right thing.